### ISSUE TITLE
Question - is it working for anyone with Android?

### ISSUE THREAD

**Issue by ygyg70** [j3k0/cordova-plugin-openwith#40]
I was trying on Android 7 and 8, noting works.
Made some changes to the code following the Android SDK documentation, now I can share both text and images.
Changes can be seen here:
https://github.com/ygyg70/cordova-plugin-openwith/blob/master/src/android/cc/fovea/openwith/Serializer.java

**Comment by cannerycoders** [comment]
i had the same experience... Was only interested in getting a shared url and the current state-of-affairs seems to favor image/clipdata.  I hacked a solution but will gladly compare it with yours.

**Comment by NiklasMerz** [comment]
Yes the current plugin is focused on images. Text would be great but needs some work.
